config/software/ruby.rb

```diff
@@ -233,6 +233,10 @@
     configure_command << "--with-opt-dir=#{install_dir}/embedded"
   end
 
+  if version == "2.3.4"
```

Comment:
might want to just make a comment above this that the patch is included in 2.3.5/2.4.1, so we know it can be removed